pkg/api/vlabs/types.go

```diff
@@ -38,6 +38,7 @@ type Properties struct {
 	WindowsProfile          *WindowsProfile          `json:"windowsProfile,omitempty"`
 	ServicePrincipalProfile *ServicePrincipalProfile `json:"servicePrincipalProfile,omitempty"`
 	CertificateProfile      *CertificateProfile      `json:"certificateProfile,omitempty"`
+	AadProfile              *AadProfile              `json:"aadProfile,omitempty"`
```

Comment:
To keep this consistent with the field naming scheme in the API Model e.g: APIServerCertificate, SSH, DNSPrefix - please rename to: "AADProfile"